ui/helpers/constants/settings.js

```diff
@@ -151,46 +151,54 @@ const SETTINGS_CONSTANTS = [
     iconName: IconName.Book,
   },
   // securityAndPrivacy settingsRefs[0]
+  {
+    tabMessage: (t) => t('securityAndPrivacy'),
+    sectionMessage: (t) => t('basicConfigurationLabel'),
+    descriptionMessage: (t) => t('basicConfigurationDescription'),
+    route: `${SECURITY_ROUTE}#basic-functionality-toggle`,
+    icon: 'fa fa-lock',
+  },
+  // securityAndPrivacy settingsRefs[1]
```

Comment:
This PR did allow me to search + jump to the basic toggle.  But I notice some other settings breaking.  Looking for fix

Before on latest develop, IPFS gateway would jump correctly:

https://github.com/MetaMask/metamask-extension/assets/3500406/1d9b6b9b-e880-459c-aa41-6757b662f6ce

But on this branch it jumps me to autodetect instead:


https://github.com/MetaMask/metamask-extension/assets/3500406/40240b7e-6977-4191-877e-d3555ea8df5f

Reply:
Ok I'm more confused, since I seem to be able to search for basic functionality even on latest develop

Reply:
I think this is the fix.  Just shifting the other indices that incremented (all except 18)

https://github.com/MetaMask/metamask-extension/commit/1dc663df37ce0c4cdbed0694cc44e914e433491b